### PR TITLE
image search and get published stories

### PR DESCRIPTION
**Pull request by ShreyasRanganatha**

### CODE REVIEW

**Review by DodonInd — APPROVED** [review]

File: api-docs-v1.md [hunk 1]
```diff
@@ -59,7 +59,6 @@
     "cards": [
```
Comment (DodonInd) [inline api-docs-v1.md]: should the name be changed to stories?

Comment (ShreyasRanganatha) [reply api-docs-v1.md]: yes

Comment (ShreyasRanganatha) [reply api-docs-v1.md]: updated

File: api-docs-v1.md [hunk 2]
```diff
@@ -56,15 +56,14 @@
      ]
     "publisherName":string,    
     "publisherId": string      
-    "cards": [ 
+    "stories": [
```
Comment (vinay8494) [inline api-docs-v1.md:59]: let it be card or pages.

File: api-docs-v1.md [hunk 3]
```diff
@@ -56,15 +56,14 @@
...
+    "stories": [ 
       { 
         "pageId" : string 
-        "imageUrl": string 
         "imageAttribution": string 
         "isCoverCard": boolean
         "summary": string, 
         "sequenceNumber": integer 
-        "url": string 
+        "videoUrl": string
```
Comment (vinay8494) [inline api-docs-v1.md:66]: keep it url as in future we can have some other urls like webpeek etc. in this.

**Pull request merged**